labgrid-web-client/src/app/_services/place.service.ts

```diff
@@ -26,6 +28,10 @@ export class PlaceService {
     }
 
     public async getPlaces(): Promise<Place[]> {
+        // If the python-wamp-client is not available the following lines can be used to load test data
+        // let mockPlaces = await this._http.get('../../assets/places.json').toPromise() as Place[]
+        // return mockPlaces 
```

Comment:
You should insert semicolons after each line of code even if it is commented out.